docs/examples/vector_stores/AstraDBIndexDemo.ipynb

```diff
@@ -0,0 +1,261 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Astra DB\n",
+    "\n",
+    ">[Astra DB](https://astra.datastax.com) is a NoSQL, row-oriented, highly scalable and highly available database.\n",
```

Comment:
I'd link to https://docs.datastax.com/en/astra/home/astra.html

Reply:
This blurb was a mistake on my part. AI folks are not interested in these details. Suggestion (to your judgement):

`DataStax Astra DB is a serverless vector-capable database built on Cassandra and accessed through an easy-to-use JSON API.`